spec/requests/stories/feeds_spec.rb

```diff
@@ -202,21 +202,16 @@
 
     context "when there are low-scoring articles" do
       let!(:article) { create(:article, featured: false) }
-      let!(:article_with_mid_score) do
-        minimum = Articles::Feeds::Latest::MINIMUM_SCORE
-        maximum = Settings::UserExperience.home_feed_minimum_score
-        score = (minimum..maximum).to_a.sample
```

Comment:
I believe this is the reason why this spec has been so flaky. The `..` is inclusive, so there's a chance that minimum score is used and this "mid-score" article will no longer be considered mid-score.

Reply:
So that things are :link:d: I stumbled into this same flaky test but wound up [fixing it in a different way](https://github.com/forem/forem/pull/19169/commits/45bf74d5c796120878f7b7316508fb69881d4599) on a branch that hasn't merged yet. If this merges first, I'll update my branch to adapt.